bundles/binding/org.openhab.binding.fritzboxtr064/src/main/java/org/openhab/binding/fritzboxtr064/internal/Tr064Comm.java

```diff
@@ -505,7 +508,23 @@ private SOAPMessage readSoapResponse(String soapActionHeader, SOAPMessage reques
 
     }
 
-    /***
+    /**
+     * In case of failure reset the authentication state, close connection and init again.
+     */
+    private void resetHttpClient() {
+    	logger.trace("Drop client for fritzbox and setup connection again.");
+		if (_httpClientContext.getTargetAuthState() != null) {
+			_httpClientContext.getTargetAuthState().reset();
+		}
+    	try {
+			_httpClient.close();
+		} catch (IOException e) {
+			logger.error("Failed to close connection to fritzbox at {}. This might result in still open, but dead connections waiting for a timeout.", _url, e);
```

Comment:
This probably should not be higher than INFO. I'd argue for DEBUG.

Reply:
:+1: We're not in any kind of high load situation where this could be critical, so I'm fine with DEBUG.